ocs_ci/ocs/ui/page_objects/overview_tab.py

```diff
@@ -22,8 +24,145 @@ def wait_storagesystem_popup(self) -> bool:
         logger.info(
             "Wait and check for Storage System under Status card on Overview page"
         )
-        return self.wait_until_expected_text_is_found(
+        is_present = self.wait_until_expected_text_is_found(
             locator=self.validation_loc["storagesystem-status-card"],
             timeout=30,
             expected_text="Storage System",
         )
+        if not is_present:
+            logger.warning(
+                "Storage System under Status card on Data Foundation Overview tab is missing"
+            )
+        return is_present
+
+    def open_storage_popup_from_status_card(self):
+        """
+        Open Storage System popup from Status card
+        """
+        logger.info("Open Storage System popup from Status card")
+        self.do_click(
+            self.validation_loc["storagesystem-status-card"], enable_screenshot=True
+        )
+
+    def validate_block_and_file_ready(self):
+        """
+        Validate Block and File Storage are Ready
+        Returns:
```

Comment:
We usually have an extra line before Args and Returns for better readability, perhaps you could add it here

Reply:
added space